helper-cli/src/main/kotlin/commands/ListCopyrightsCommand.kt

```diff
@@ -71,7 +71,7 @@ internal class ListCopyrightsCommand : CliktCommand(
         help = "Show the raw statements corresponding to each processed statement if these are any different."
     ).flag()
 
-    private val packageConfigurationOption by mutuallyExclusiveOptions<PackageConfigurationOption>(
+    private val packageConfigurationOption by mutuallyExclusiveOptions(
```

Comment:
How about squashing this to the previous commit and changing the title to "(helper-)cli: ..."?

Reply:
Done.